app/routes/questionnaire.py

```diff
@@ -544,14 +546,28 @@ def get_page_title_for_location(schema, current_location, context):
     return safe_content(page_title)
 
 
-def _render_page(block_type, context, current_location, previous_location_url, schema):
+def _render_page(
+    block_type,
+    context,
+    current_location,
+    previous_location_url,
+    schema,
+    questionnaire_store,
+):
     if request_wants_json():
         return jsonify(context)
 
     page_title = get_page_title_for_location(schema, current_location, context)
     session_data = get_session_store().session_data
     session_timeout = get_session_timeout_in_seconds(schema)
 
+    hub_required_sections = all(
```

Comment:
It makes sense if this var was `are_hub_required_sections_complete`. This is also used in more than 1 place now, so might make sense to chuck it in a method.